src/Controls/src/Core/Page/Page.cs

```diff
@@ -304,16 +403,27 @@ protected virtual void LayoutChildren(double x, double y, double width, double h
 			}
 		}
 
+		/// <summary>
+		/// When overridden in a derived class, allows application developers to customize behavior immediately prior to the page becoming visible.
+		/// </summary>
 		protected virtual void OnAppearing()
 		{
 		}
 
+		/// <summary>
+		/// Determines the behavior when the back button of the page is pressed.
+		/// </summary>
+		/// <returns><see langword="true"/> when the back navigation was handled by the override, otherwise <see langword="false"/>.</returns>
+		/// <remarks>
+		/// <para>Application developers can override this method to provide behavior when the back button is pressed. 
+		/// When overridden to handle or cancel the navigation yourself, this method should return <see langword="true"/>.</para>
+		/// <para>This only works on Android and UWP for the hardware back-button. On iOS, this method will never be called because there is no hardware back button.</para>
```

Comment:
```suggestion
		/// <para>This only works on Android and UWP for the hardware back button. On iOS, this method will never be called because there is no hardware back button.</para>
```